Makefile

```
@@ -9,7 +11,7 @@ build: ## build the docker version
 	docker build -t $(DOCKER_PREFIX)/$(DOCKER_IMAGE) .
 
 run: ## run the latest docker build
-	docker run --rm -it --name $(DOCKER_IMAGE) -p 3000:3000 -t $(DOCKER_PREFIX)/$(DOCKER_IMAGE)
+	docker run --rm -it --name $(DOCKER_IMAGE) -e CONFLUENCE_USERNAME=$(CONFLUENCE_USERNAME) -e CONFLUENCE_PASSWORD=$(CONFLUENCE_PASSWORD) -p 3000:3000 -t $(DOCKER_PREFIX)/$(DOCKER_IMAGE)
```

```suggestion
 	docker run --rm -it --name $(DOCKER_IMAGE) -e CONFLUENCE_USERNAME -e CONFLUENCE_PASSWORD -p 3000:3000 -t $(DOCKER_PREFIX)/$(DOCKER_IMAGE)
```

Prevents worrying about escaping passwords, and prevents it from showing up in your shell/ps/etc

Good practice